frontend/src/lib/Utils.tsx

```diff
@@ -134,6 +142,12 @@ export function getRunDurationFromApiRun(apiRun?: ApiRun): string {
   return getDuration(new Date(apiRun.created_at), new Date(apiRun.finished_at));
 }
 
+export function getRunDurationFromRunV2(run?: V2beta1Run): string {
```

Comment:
It looks duplicated with getRunDurationV2() above

Reply:
Great catch. Will remove this one.